core/src/commonMain/kotlin/com/xebia/functional/xef/auto/AIRuntime.kt

```diff
@@ -13,9 +15,14 @@ data class AIRuntime<A>(val runtime: suspend (block: AI<A>) -> A) {
     @OptIn(ExperimentalTime::class)
     @JvmStatic
     fun <A> openAI(): AIRuntime<A> = AIRuntime { block ->
-      val openAIConfig = OpenAIConfig()
-      KtorOpenAIClient(openAIConfig).use { openAiClient ->
-        val embeddings = OpenAIEmbeddings(openAIConfig, openAiClient)
+      val openAIConfig =
+        OpenAIConfig(
```

Comment:
Now we are using the `OpenAIConfig` from `com.aallam.openai` I guess we can delete [our domain's](https://github.com/xebia-functional/xef/blob/open-ai-typed-models/core/src/commonMain/kotlin/com/xebia/functional/xef/env/config.kt#L5):

Reply:
Removed in [76d0e9e](https://github.com/xebia-functional/xef/pull/196/commits/76d0e9ee0b87c8e10a8f7e2249397c00907cec5f)